tests/index.test.ts

```diff
@@ -200,6 +191,14 @@ describe("formatError", () => {
                 result: "Error message: Oops: something is wrong",
             });
         });
+
+        it("if the error message doesn't contain contract address", () => {
+            expect(formatError(formatErrorTestData[11])).toEqual({
+                protocol: undefined,
+                interface: undefined,
+                result: "Error message default: Oops: something is wrong",
```

Comment:
I'm not sure about the `Error message **default**` prefix

Reply:
This message is only used in the tests configuration json file to make sure this case is handled by the default handler. Real users will never see this message. In production, different JSON file is used.